### ISSUE TITLE
Create Icon for Linux App Running in Chromebook

### ISSUE THREAD

**Issue by trystant** [trystant/knowledgebase#82]
I currently use [Firefox Developer Edition](https://www.mozilla.org/en-US/firefox/developer/) in the default Linux container on my PixelBook.  I know that there's a way to create an icon for any application on the container; I've done it in the past and need to figure out how to do that again.

**Comment by trystant** [comment]
Having done some additional research, it looks like this [thread](https://support.google.com/pixelbook/thread/708306?hl=en) on the Pixelbook support forums describes the files used to make those desktop icons (.desktop files) as well as where & how to place them on one's local system (`~/.local/share/applications` is where the `.desktop` files go). I found a file that had been renamed in the past to something that couldn't be picked up. I'm not sure why that is the case, but I was able to modify that `.desktop` file and now the icon is showing up in Pixelbook app results properly. 
![image](https://user-images.githubusercontent.com/13509/103473091-ad074280-4d62-11eb-920b-4b407a4ab73a.png).  This resolves the bug at hand.
